explorer/src/Pos/Explorer/Web/Api.hs

```diff
@@ -37,8 +38,9 @@ type PageNumber = Integer
 type API = "api"
 
 type TotalAda = API
-    :> "totalada"
-    :> Get '[JSON] (Either ExplorerError CCoin)
+    :> "supply"
+    :> "ada"
+    :> Get '[JSON] (Either ExplorerError Micro)
```

Comment:
:neutral_face: I won't say it's great, but I'm not going to be a review Nazi either.

Reply:
Could someone explain me why `CCoin` was used here? I know that it is used in wallet to pass number through daedalus-bridge as string to prevent precision losses. But here we do not have bridge, right?

Reply:
Explorer frontend possibly? We do have a "bridge" for _explorer_, we are using `CCoin` on other endpoints (in the data structure, though).

Reply:
If we have bridge for explorer, then I guess that `Mirco` is not ok here :/

Reply:
@Martoon-00 bridge is not using this endpoint (at least not atm). I am not sure how Micro is serialized. Will test it.

Reply:
But there is a point to what @Martoon-00 is saying. If we use this endpoint from _explorer_ do we have the corresponding client representation?

Reply:
@ksaric Micro is serializable somehow , I just didn't check it myself to see how it serializes things. The best would be just to return a number (or json alternatively).

I am currently compiling in order to check it